src/providers/deepinfra/api.ts

```diff
@@ -0,0 +1,14 @@
+import { ProviderAPIConfig } from "../types";
+
+const DeepInfraApiConfig: ProviderAPIConfig = {
+  baseURL: "https://api.deepinfra.com/v1/openai",
+  headers: (API_KEY: string) => {
+    return {
+      Authorization: `Bearer ${API_KEY}`,
+      "Content-Type": "application/json",
```

Comment:
We already add content-type header in the below function. So we can remove it from here. Otherwise, this will cause duplicate header
https://github.com/Portkey-AI/gateway/blob/b5dbfadd5bc697c5c212cf44a3bc8bea898f3216/src/handlers/handlerUtils.ts#L20-L23